web-api/src/business/useCases/auth/signUpUserInteractor.ts

```diff
@@ -25,15 +22,15 @@ export const signUpUserInteractor = async (
     };
   },
 ): Promise<SignUpUserResponse> => {
-  const cognito: CognitoIdentityProvider = applicationContext.getCognito();
-
   // Temporary code to prevent creation of duplicate accounts while Cognito is still case sensitive.
   // We do not want to allow people to make two accounts for the same email that only differ by casing.
-  const { Users: existingAccounts } = await cognito.listUsers({
-    AttributesToGet: ['email'],
-    Filter: `email = "${user.email}"`,
-    UserPoolId: applicationContext.environment.userPoolId,
-  });
+  const { Users: existingAccounts } = await applicationContext
+    .getCognito()
+    .listUsers({
+      AttributesToGet: ['email'],
+      Filter: `email = "${user.email}"`,
+      UserPoolId: applicationContext.environment.userPoolId,
+    });
```

Comment:
Seems like this should be behind the UserGateway as well?

Reply:
You are correct!

It has since been moved as a part of 10208. I did this refactoring solo and needed an ensemble for more help to refactor this one. Along with `changePasswordInteractor`. Is that OK? I can also try to cherry-pick those changes onto this branch if you would prefer.